### ISSUE TITLE
Long forgotten main files

### ISSUE THREAD

**Issue by jimaek** [jsdelivr/bot#32]
What can we do to make these errors disappear https://github.com/jsdelivr/jsdelivr/pull/3601 ? We created a custom mainfile a long time ago so there is no point showing these errors for ever

**Comment by megawac** [comment]
We could add a field to the info.ini?

**Comment by jimaek** [comment]
What kind of field? I guess we need to add .mainfile to the API for this feature to work?

**Comment by megawac** [comment]
Maybe something like

`checkMainFiles: > 1.2`

**Comment by jimaek** [comment]
Nah, too much work to maintain. I think we need to add it to the API, then your bot could simply parse it and only throw errors when there is no `mainfile` override file. What do you think?

**Comment by jimaek** [comment]
@megawac Our API now supports this http://api.jsdelivr.com/v1/jsdelivr/libraries?name=imagesloaded
Each version has its own mainfile parameter as well.

**Comment by megawac** [comment]
The main difficulty here is supporting both `.mainfile` (#10) and this. For instance if the user changes the mainfile (https://github.com/jsdelivr/jsdelivr/pull/5927), its difficult to determine if that affects a folder with a `.mainfile`. One possible workaround would be to have the API include a flag for each version if a `.mainfile` is set

**Comment by jimaek** [comment]
@megawac Can we include this feature into v2? Can you make sure we dont forget this?

**Comment by megawac** [comment]
This can also be added to v1. Either way, it should be a simple fix

**Comment by jimaek** [comment]
Lets add it only to v2, there has to be motivation to switch to v2.
